defi/src/storeGetProtocols.ts

```diff
@@ -90,7 +90,7 @@ export async function storeGetProtocols({
         };
       })
     )
-  ).filter((p) => p.category !== "Chain" && p.category !== "CEX");
+  ).filter((p) => !["Chain", "CEX", 'Canonical Bridge'].includes(p.category ?? ""));
```

Comment:
hmm, can you turn this into a set and add it to this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/utils/excludeProtocols.ts with a comment explaining the reason for the variable.

I am guessing the reason is, these are categories hidden from protocol list/wont show up in search.

same change for line 182.


this way, all category lists are maintained in the same file, when we add a new category, we wont forget to add it to appropriate lists